Localization/en-US/statistics.po

```diff
@@ -0,0 +1,50 @@
+# Schema-Version: 1.0
+# URL: https://foldingathome.org/statistics/
+# Source-Language: en-US
+# Source-Version: 1.0
+# Source-Last-Modified: 20200705
+# Target-Language:
+# Target-Version:
+# Target-Last_Modified:
+
+msgid "Statistics"
+msgstr ""
+
+msgid "One of the best ways to help Folding@home is by recruiting your friends and family. Start by sharing our project with them. Then join a team or even start your own team. The more points your team earns, the closer we come to finding cures."
+msgstr ""
+
+msgid "On this page you will find access to statistics for individuals and teams who have joined together to earn points and compete with other teams. Some of us are quite intense in our approach to folding. We have team websites, we supe up our computers, and we drive the technology forward by reporting bugs and making suggestions about how to improve the software."
+msgstr ""
+
+msgid "MAXIMIZE"
+msgstr ""
+
+msgid "your"
+msgstr ""
+
+msgid "**EFFORT**"
+msgstr ""
```

Comment:
This (and the next two) violate the idea of keeping segments together as much as possible. But we would have to ask the designers to change the way they set this up, e.g. by putting the fancy formatting on the `<strong>` tag so that we can do something like this here:
```suggestion
msgid "**MAXIMIZE** your **EFFORT**"
msgstr ""
```